src/supabase/supabase_manager.py

```
@@ -587,17 +591,167 @@ def upload_all(self, tables: List[str] = None,
+        token = self._get_management_api_token()
+        if not token:
+            # Fallback to PostgREST probe
+            return self._get_supabase_columns_via_client(table_name)
+
+        ref = self._get_project_ref()
+        query = (
+            f"SELECT column_name FROM information_schema.columns "
+            f"WHERE table_schema = 'public' AND table_name = '{table_name}' "
+            f"ORDER BY ordinal_position;"
+        )
+        resp = requests.post(
```

Ruff (0.15.2)

[error] Probable use of `requests` call without timeout

(S113)